examples/overview/build/index.main.mjs

```diff
@@ -5,6 +5,7 @@ export const _version = '0.1.2';
 
```

Comment:
I try not to commit these files if nothing interesting is different, because it creates noise in the history